extras/docker/fromsource/heketi-start.sh

```diff
@@ -1,5 +1,50 @@
 #!/bin/sh
 
+: ${HEKETI_PATH:=/var/lib/heketi}
```

Comment:
Could you add variables for `${HEKETI_PATH}/heketi.db` and `${HEKETI_PATH}/container.log` ? To reduce the amount of typing and  chance for typos.